sql/core/src/test/resources/sql-tests/inputs/group-by.sql

```diff
@@ -53,3 +53,13 @@ set spark.sql.groupByAliases=false;
 
 -- Check analysis exceptions
 SELECT a AS k, COUNT(b) FROM testData GROUP BY k;
+
+-- turn on group by aliases (for remaining tests)
+set spark.sql.groupByAliases=true;
```

Comment:
why do we need to turn on `groupByAliases`? seems following tests don't use this feature